src/libtorch.cc

```diff
@@ -552,11 +582,48 @@ ModelInstanceState::Create(
 
 ModelInstanceState::ModelInstanceState(
     ModelState* model_state, TRITONBACKEND_ModelInstance* triton_model_instance)
+#ifdef TRITON_ENABLE_GPU
+    : BackendModelInstance(model_state, triton_model_instance),
+      model_state_(model_state), device_(torch::kCPU), is_dict_input_(false),
+      pytorch_stream_(at::cuda::getDefaultCUDAStream())
+#else
     : BackendModelInstance(model_state, triton_model_instance),
       model_state_(model_state), device_(torch::kCPU), is_dict_input_(false)
+#endif
```

Comment:
Change to below? Or initialize `pytorch_stream_` in constructor body

```suggestion
    : BackendModelInstance(model_state, triton_model_instance),
      model_state_(model_state), device_(torch::kCPU), is_dict_input_(false)
#ifdef TRITON_ENABLE_GPU
      , pytorch_stream_(at::cuda::getDefaultCUDAStream())
#endif
```

Reply:
The problem is that `pytorch_stream_` is not default constructible. Because of this it must be initialized in the initializer list. I also tried using a unique_ptr or a pointer to the return argument of `at::cuda::getDefaultCUDAStream()`. However, it is an r-value and you cannot get the pointer to an r-value. Let me know if you have any other ideas.

Reply:
I have fixed this in the next commit.